content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+deploy the workload automatically. It removes the need to exec into the node or configure
+VMSS extensions to perform these administrative actions. You get all the benefits of
+containerized packaging and deployment methods combined with a reduction in both administrative
+and development cost.
+
+HostProcess containers differ quite significantly from regular Windows Server containers.
+They are run directly as processes on the host under the access policies of
+a user you specify. HostProcess containers run as either the built-in system accounts or
+ephemeral users within a user group defined by you. HostProcess containers also share
+the host's network namespace and access/configure storage mounts visible to the host.
+On the other hand, Windows Server containers are highly isolated and exist in a separate
+silo'd namespace. Direct access to the host from a Windows Server container is explicitly disallowed
+by default. Mounting a host volume into the Windows Server container is highly discouraged to minimize
+potential attack surfaces. HostProcess containers enable you to configure the host such that communication
+with your Windows container workloads remains within the bounds of a secure pipeline.
+
+Until now, scenarios common to Linux privileged containers, such as kube-proxy (via kubeadm),
+storage, and networking, all required proxy solutions to enable functionality on Windows.
+With HostProcess containers users have direct access to the node without the need for
+any workarounds. Gone are the days where cluster operators would need to manually log onto
+Windows nodes to perform administrative duties - instead replaced with a continuous
+development pipeline.
+
+## How does it work?
+
+Windows HostProcess containers are implemented with Windows _Job Objects_, a break from the
+previous container model using server silos. Job objects are components of the Windows OS which offer the ability to
+manage a group of processes as a group (a.k.a. _jobs_) and assign resource constraints to the
+group as a whole. Job objects are specific to the Windows OS and are not associated with the Kubernetes [Job API](https://kubernetes.io/docs/concepts/workloads/controllers/job/). They have no process or file system isolation,
+enabling the privileged payload to view and edit the host file system with the
+correct permissions, among other host resources. The init process, and any processes
+it launches or that are explicitly launched by the user, are all assigned to the
+job object of that container. When the init process exits or is signaled to exit,
+all the processes in the job will be signaled to exit, the job handle will be
+closed and the storage will be unmounted.
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own pod security policies. Those used to configure Linux privileged
```

Comment:
Pod security policies? Not in Kubernetes 1.26 they don't. Maybe write “Pod-level security restrictions”

Reply:
and maybe link to https://kubernetes.io/docs/concepts/security/pod-security-admission/?

Reply:
I was thinking the same thing. Will update after reviewing suggested changed